included description of setFeature function (#4187)
qgis/QGIS-Documentation: source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst

  override` button, to provide a URL from the contents of an attribute field of
  a table or using a regular expression.
* In :guilabel:`Source` you can enter text in the textbox with some HTML tags or

>   provide a full HTML page. Provided that you created a relation described in

Reviewer: I'd skip all the verbal reference to the section with adding a hyperlink
```suggestion
  provide a full HTML page. Provided that you created a :ref:`relation between
  layers <vector_relations>` a simple code including
```

>   in :ref:`Creating one or many to many relations<_vector_relations>` a simple code including

Reviewer: ```suggestion
```

  HTML, JavaScript and GeoJSON gives you full access to the parent and child features. 

>   This way, the atlas feature browses through the parent layers and and creates a page

Reviewer: ```suggestion
  This way, the atlas feature browses through the parent layers and creates a page
```

  for every existing relation, depicting a parent feature and the relating child features

>   on one page each.

Reviewer: not sure I understand what happens

Author: @DelazJ I added more code and tried to describe what happens in there !

* In :guilabel:`Source` you can enter text in the textbox with some HTML tags or
  provide a full HTML page. Provided that you created a :ref:`relation between
  layers <vector_relations>` a simple code including
  HTML, JavaScript and GeoJSON gives you full access to the parent and child features. 
  The following code requires an existing relation ``my_relation`` and creates a dynamic bullet
  list of related child features.
  
  .. code-block:: HTML 
  
     <script>
     <div id="name"></div>
     <div id="my_children"></div>
     
     <script type="text/javascript">
      function setFeature(feature)
               {
                document.getElementById('name').innerHTML = feature.properties.NAME;
                document.getElementById('my_children').innerHTML = '';
                feature.properties.my_relation.forEach(function(child_feature)
                                       {
                                       var node = document.createElement("li"):
                                       node.appendChild(document.createTextNode(child_feature.NAME));
                                       document.getElementById('my_children').appendChild(node);

>                                        }

Reviewer 2: Missing end parenthesis for "forEach"?
